### ISSUE TITLE
Status bar item hover potential accessibility improvements

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#127415]
Refs: https://github.com/microsoft/vscode/issues/127372

Status bar items hovers are in general not accessible. However we solve that for our internal items by each item having an area label that is sort of a combination of the content and the hover, so the screen reader users would know what is going on.

For extension contributed `statusBarItems` this can be controlled via `myStatusBarItem.accessibilityInformation` which is nice.
However if the `accessibilityInformation` is not provided we could:
* append the tooltip to the the current aria label (so it is at the end since not so important)

Or in general we could introduce a command
* show hover (ctrl/cmd + k, ctrl/cmd + i) which shows the status bar item hover and passes dom focus into the hover. That way this would be aligned with the hover behaviour in editor and tree (when hover triggered via keyboard pass focus into it)

This is not a big issue, but wanted to share some ideas.

**Comment by bpasero** [comment]
To clarify, this is only an issue for extension provided hovers that do not provide accessibility info? Or is there more?

**Comment by bpasero** [comment]
@isidorn it looks like you have been computing a custom ARIA label when you introduced the accessibility information API:

https://github.com/microsoft/vscode/commit/6e50f24e64dd085e373376c381253ead02069ea1#diff-039b1e0146efcb67b6184b85efeb2c5722c86c3b7da3d584b81d8f52029310d4R35

Maybe you can do the change you propose to add more info to the accessibility label if it is not provided. We recently added a `name` property for status bar items which maybe can be used as prefix too for the ARIA label?

**Comment by isidorn** [comment]
I am now appending the title to the end of the `ariaLabel` of a `statusBar` item if the `accessibilityInformation` is not provided.
I decided to not add a name to not make the aria label too verbose.

**Comment by lramos15** [comment]
Should I be using the dev tools to inspect the aria label of a contributed status bar item?

**Comment by isidorn** [comment]
@lramos15 yeah a dev tools inspect is fine. Or just turn on screen reader, F1 > focus status bar > move focus around using the keyboard and make sure the screen reader reads appropriate descriptions for status bar items
